package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "cover": "nyc --reporter=lcov --reporter=text --reporter=html npm run tests-only",
-    "lint": "eslint --ext .js,.json .",
+    "lint": "eslint .",
```

Comment:
```suggestion
    "lint": "eslint --ext=.js,.json,.mjs,.cjs .",
```

Reply:
I believe the `--ext` option is obsolete in flat config. In fact, the following error occurs.

```
$ npm run lint

> eslint-find-rules@4.2.0 lint
> eslint --ext=.js,.json,.mjs,.cjs .

Invalid option '--ext' - perhaps you meant '-c'?
You're using eslint.config.js, some command line flags are no longer available. Please see https://eslint.org/docs/latest/use/command-line-interface for details.
```

Therefore, the `files` option set.
![image](https://github.com/user-attachments/assets/f6745d4f-9fae-473c-84d6-1ad95a65a94e)

Reply:
ah, fair enough.